src/Microsoft.VisualStudio.Services.Agent/Util/NetFrameworkUtil.cs

```diff
@@ -154,9 +154,13 @@ private void InitVersions()
                             {
                                 versionString = "4.6.1";
                             }
-                            else if (release > 394271)
+                            else if (release > 394271 && release <= 460797)
                             {
-                                versionString = "4.6.2";
+                                versionString = "4.6.1";
```

Comment:
shouldn't this be 4.6.2?

Reply:
You're dang right it should! Thanks for looking out!